hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutBackupOperation.java

```diff
@@ -37,7 +41,7 @@
     private boolean unlockKey;
     private RecordInfo recordInfo;
     private boolean putTransient;
-
+    private boolean wanOriginated;
```

Comment:
It means that this backup operation is backup of `WanOriginatedMergeOperation`. Since forwarding of incoming wan events to other clusters are done in related `ReplicationSupportingService`,  `WanOriginatedMergeOperation` and its backup should not publish the event again.